### PR TITLE
demo commit

### PR DESCRIPTION
**Pull request by imazik**
This PR is created to give a session on how to send a pr in github. Will close this soon.
Signed-off-by: Shovan Maity <shovan.cse91@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,4 +60,4 @@ the projects used by the Litmus project may be governed by a different license,
 <a href="https://landscape.cncf.io/selected=litmus">
   CNCF Landscape <img src="https://landscape.cncf.io/images/left-logo.svg" alt="Litmus on CNCF Landscape" height="15">
 </a>
-
+something new
```
Comment (imazik) [inline README.md]: please change this

**Comment by rajdas98** [comment]
demo review

**Comment by rajdas98** [comment]
look good to me

**Comment by imazik** [comment]
This PR was created to give a session on how to send a pr in github closing this.

**Pull request closed without merging**